<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QDialog" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>充值</string>
  </property>
  <widget class="QLabel" name="lbQRCode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;h1&gt;二维码&lt;/h1&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="btnAlipay">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>支付宝</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">bgPayMethods</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="btnWechatPay">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>115</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>微信</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">bgPayMethods</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btnGetCode">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>获取二维码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAmount">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>充值金额：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bgPayMethods"/>
 </buttongroups>
</ui>
